src/main/scala/SqliteDb.scala

```diff
@@ -77,15 +77,15 @@ class SqliteResultIterator(db: SqliteDb, private val stmt: Long)
                         case Sqlite3C.TEXT => SqlText(new String(Sqlite3C.column_blob(stmt, i)))
                         case Sqlite3C.BLOB => SqlBlob(Sqlite3C.column_blob(stmt, i))
                         case Sqlite3C.NULL => SqlNull
-                        case _ => sys.error("unsupported type")
+                        case _ => throw new SqlException("unsupported type")
```

Comment:
I don't think you need the `new` since it's now a case class, but I don't have a preference for or against using it